Docs/user_guide/modules/annotations.md

```diff
@@ -1,8 +1,14 @@
 # Annotations
 
+:::{warning}
+
+This module is deprecated and will be removed in the future. It is recommended to use the [Markups](markups.md) module.

```

Comment:
Documentation  links are currently specified with and without the `.md` suffix.

Being explicit makes sense and may help reduce confusion when writing documentation.

```bash
$ cd Slicer/Docs
for file in $(ls -1 user_guide/modules/); do \
  name=$(echo $file | cut -d"." -f1); \
  ack -hi "\[\w+\s*\]\(\w*"$name; \
  ack -hi "\[\w+\s*\]\(modules\/\w*"$name; \
done
```

![image](https://user-images.githubusercontent.com/219043/143468860-f5d4801c-0b08-4c7a-b1be-05d1279ee1e1.png)

Reply:
I've pushed a commit with changes to explicitly include the .md suffix when linking module pages. See if I fixed up everyplace correctly.